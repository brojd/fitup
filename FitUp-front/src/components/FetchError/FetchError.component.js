import React from 'react';

const FetchError = () => ((
  <div className='uk-alert uk-alert-danger uk-width-5-10 uk-align-center uk-text-center uk-margin-large-top'>
    Sorry, cannot load data from server...
  </div>
));

export default FetchError;
