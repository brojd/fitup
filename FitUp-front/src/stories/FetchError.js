import React from 'react';
import { storiesOf} from '@kadira/storybook';
import FetchError from '../components/FetchError/FetchError.component';

storiesOf('FetchError', module)
  .add('fetch error message', () => (
    <FetchError />
  ));
