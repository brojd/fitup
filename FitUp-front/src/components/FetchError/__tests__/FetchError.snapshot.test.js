import React from 'react';
import FetchError from '../FetchError.component';
import renderer from 'react-test-renderer';

test('FetchError renders correctly', () => {
  const component = renderer.create(
    <FetchError />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
