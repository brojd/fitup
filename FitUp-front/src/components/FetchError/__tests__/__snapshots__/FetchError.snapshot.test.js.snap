exports[`test FetchError renders correctly 1`] = `
<div
  className="uk-alert uk-alert-danger uk-width-5-10 uk-align-center uk-text-center uk-margin-large-top">
  Sorry, cannot load data from server...
</div>
`;
